README.md

```diff
@@ -178,6 +178,22 @@ Contour fails fast and loud when these errors are detected, and provides as much
   <img src="screenshots/crd.png">
 </p>
 
+## Releases
+```groovy
+implementation 'app.cash.contour:contour:(latest)'
+```
+
+Snapshots of the development version are available in [Sonatype's `snapshots` repository](snap).
```

Comment:
I noticed the same on Paparazzi but this url resolves to `https://github.com/cashapp/contour/blob/master/snap`

Reply:
ooof.  looks like Butterknife uses [] instead of ().  Trying that.

Reply:
nice